app/Models/NetworkBanner.php

```diff
@@ -36,6 +39,12 @@
  */
 class NetworkBanner extends Model
 {
+    public const HTML_TYPE = 'html';
```

Comment:
`TYPE_HTML` is more readable when we have more types

Reply:
changed